packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -95,7 +95,6 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, any> {
                   iconProps={ { iconName: 'More' } }
                   role='button'
                   aria-haspopup='true'
-                  onRenderMenuIcon={ () => null }
```

Comment:
This is a change in behavior here. We need to pass in a function that returns null or false. 

You could just define this at the top:

```tsx
const falseFunction = () => false;
```

Reply:
@dzearing Fixed (I returned `null` because `IRenderFunction` expects `JSX.Element` or `null`)

Reply:
@dzearing Are you seeing lines 27 and 99 that were added? 

https://github.com/OfficeDev/office-ui-fabric-react/pull/2814/commits/62c141280c85e110d7f9212f0af26bb7a2fa87a0